### PR TITLE
3uTools.3uTools version 2.63

### PR DESCRIPTION
**Pull request by Marlon5775**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/82095)

### CODE REVIEW

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://url.3u.com/zmAJjyaa
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml:9]:
```suggestion
  InstallerUrl: https://d-updater.3u.com/3utools/download/3uTools_v2.63.003_Setup.exe
```

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://url.3u.com/zmAJjyaa
+  InstallerSha256: 230A196E56C37F9B1AB3451A27D4CF435202741A35E5E805622628E2274011ED
+  InstallerSwitches:
+    Silent: 0
+    SilentWithProgress: 0
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml:13]: These switches do not work to install the application silently
image

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.63.003
```

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.installer.yaml:5]:
```suggestion
PackageVersion: 2.63.003
```

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.yaml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.yaml:5]:
```suggestion
PackageVersion: 2.63.003
```

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
+PackageLocale: en-US
+Publisher: 3uTools
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml:7]:
```suggestion
Publisher: ShangHai ZhangZheng Network Technology Co., Ltd.
```

File: manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: 3uTools.3uTools
+PackageVersion: 2.63
+PackageLocale: en-US
+Publisher: 3uTools
+PackageName: 3uTools
+License: OHNE-Lizenz
```
Comment (Trenly) [inline manifests/3/3uTools/3uTools/2.63/3uTools.3uTools.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: See above;

**Comment by SpecterShell** [comment]
This custom installer doesn't read any command line, so it cannot be installed silently.

BTW, this seems to be the international version of [i4Tools](https://www.i4.cn/), which doesn't parse command line as well.

**Pull request closed without merging**